src/validation/index.js

```diff
@@ -121,6 +123,25 @@ class Validate {
     }
     next();
   }
+
+  /**
+  * @param {object} req - Request object
+  * @param {object} res - Response object
+  * @param {callback} next - The callback that passes the request to the next handler
+  * @returns {object} res - Response object when query is invalid
+  * @memberof Validate
+  */
+  static validateMulticity(req, res, next) {
+    const { destination, flightDate } = req.body;
+    if (!destination || typeof (destination) !== 'object') {
+      return HelperMethods.clientError(res, 'destination as to be more than one');
```

Comment:
What if it is a single trip? Don't you want to say `should be at least one destination`?

Reply:
This middleware only validates multicity route

Reply:
okay